playbooks/prepare_host.yaml

```diff
@@ -69,3 +69,39 @@
     user:
       name: stack
       groups: wheel
+  - name: Prepare host for Ceph
+    when: ceph_enabled
+    block:
+    - name: Make sure we have losetup installed/latest
+      package:
+        name:
+          - util-linux
+          - lvm2
+        state: latest
+    - name: Use dd and losetup to create the loop devices
+      shell: |
+        dd if=/dev/zero of=/var/lib/ceph-osd.img bs=1 count=0 seek={{ ceph_loop_device_size }}G
```

Comment:
Suggestion:

```
truncate -s {{ ceph_loop_device_size}}G /var/lib/ceph-osd.img
```

This will create a sparse file, so it can be as big as you like as long as you don't actually try to use all of it 😀 If we decided not to use sparse:

```
fallocate -l {{ ceph_loop_device_size * <make it bytes> }} /var/lib/ceph-osd.img
```

This will preallocate the space and do lazy zeroing if the filesystem supports it (which it will), so it's very fast.